lib/webauthn/relying_party.rb

```diff
@@ -39,10 +41,18 @@ def initialize(
       @acceptable_attestation_types = acceptable_attestation_types
       @legacy_u2f_appid = legacy_u2f_appid
       self.attestation_root_certificates_finders = attestation_root_certificates_finders
+
+      if allowed_origins.nil? && !origin.nil?
```

Comment:
If you do:

```ruby
WebAuthn.configure do |config|
  config.origin = "https://auth.example.com"
end
```

as the it's suggested by the documentation, you won't get a warning since [`WebAuthn.configure` initializes an instance of `WebAuthn::Configuration`](https://github.com/cedarcode/webauthn-ruby/blob/b0903000ec431bddd4af653ad87e89064e9bd37c/lib/webauthn/configuration.rb#L8) which [initializes an instance of `RelyingParty`](https://github.com/cedarcode/webauthn-ruby/blob/b0903000ec431bddd4af653ad87e89064e9bd37c/lib/webauthn/configuration.rb#L45) and then [delegates the accessors to the relying party](https://github.com/cedarcode/webauthn-ruby/blob/b0903000ec431bddd4af653ad87e89064e9bd37c/lib/webauthn/configuration.rb#L18-L40).

```
irb(main):002> WebAuthn.configure { |config| config.origin = 'https://auth.example.com' }
=> "https://auth.example.com"
irb(main):004> WebAuthn::RelyingParty.new(origin: 'https://auth.example.com')
DEPRECATION WARNING: `WebAuthn.origin` is deprecated and will be removed in future Please use `WebAuthn.allowed_origins` instead that also allows configuring multiple origins per Relying Party
=>
#<WebAuthn::RelyingParty:0x000000011cd28ad0
 @acceptable_attestation_types=["None", "Self", "Basic", "AttCA", "Basic_or_AttCA", "AnonCA"],
 @algorithms=["ES256", "PS256", "RS256"],
 @allowed_origins=nil,
 @attestation_root_certificates_finders=[],
 @credential_options_timeout=120000,
 @encoding=:base64url,
 @id=nil,
 @legacy_u2f_appid=nil,
 @name=nil,
 @origin="https://auth.example.com",
 @silent_authentication=false,
 @verify_attestation_statement=true>
 ```

Reply:
I'll take care of that, thanks